src/store/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-cycle */
```

Comment:
we are not going to have cycle dependencies in general, so please, disable it only near the imports that causing it, not for the whole file